quail/processes/wps_climdex_sdii.py

```diff
@@ -1,19 +1,14 @@
 import os
 from rpy2 import robjects
-from pywps import Process, LiteralInput
+from pywps import Process
 from pywps.app.Common import Metadata
 from pywps.app.exceptions import ProcessError
 from rpy2.rinterface_lib.embedded import RRuntimeError
 
 from wps_tools.logging import log_handler, common_status_percentages
 from wps_tools.io import log_level, collect_args, rda_output, vector_name
-from wps_tools.R import (
-    get_package,
-    load_rdata_to_python,
-    save_python_to_rdata,
-    r_valid_name,
-)
-from quail.utils import logger, load_ci
+from wps_tools.R import get_package, save_python_to_rdata, r_valid_name
+from quail.utils import logger, load_ci, collect_literal_inputs
 from quail.io import climdex_input, ci_name, output_file
```

Comment:
I think we may be getting close to the point where we want to simplify our importing. Just something to keep in mind as our library our tools continues to grow.